WineCellar/WineCellar.GUI/DetailedView.xaml.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Controller;
+using Model;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using WineCellar.ControllerTest.Utilities;
+
+namespace WineCellar
+{
+    /// <summary>
+    /// Interaction logic for DetailedView.xaml
+    /// </summary>
+    public partial class DetailedView : Window
+    {
+        private MainWindow mainWindow;
+
+        public int IndexID { get; set; }
+
+        public DetailedView(int value)
+        {
+            InitializeComponent();
+
+            SetData(value);
```

Comment:
Gebruik hiervoor de event "Window_Loaded", await vervolgens deze methode. Momenteel is dit een async methode, die niet async uitgevoerd wordt.